src/pytorch/neighbors/getNeighborPairs.py

```diff
@@ -44,7 +49,19 @@ def getNeighborPairs(positions: Tensor, cutoff: float, max_num_neighbors: int =
         The vectors defining the periodic box.  This must have shape `(3, 3)`,
         where `box_vectors[0] = a`, `box_vectors[1] = b`, and `box_vectors[2] = c`.
         If this is omitted, periodic boundary conditions are not applied.
-
+    check_errors: bool, optional
+        If set to True the function does not throw due to a number of pairs larger than the maximum.
```

Comment:
In Python, exceptions are raised not thrown. Also, could you document what exception is raised?